rules/no-for-loop.js

```diff
@@ -304,9 +323,15 @@ const create = context => {
 							node.init.range[0],
 							node.update.range[1]
 						], replacement),
-
-						fixer.removeRange(getRemovalRange(elementVariableDeclaration, sourceCode))
-					];
+						...arrayReferences.map(reference => {
+							if (reference === elementReference) {
+								return null;
```

Comment:
```suggestion
								return;
```

Reply:
This is caught by XO:

`329:9   Arrow function expected a return value.  array-callback-return`

Reply:
Right, you can `return undefined;` then.

Reply:
750aa1c